openverse_catalog/dags/maintenance/pr_review_reminders.py

```diff
@@ -0,0 +1,60 @@
+"""
+Iterates through open PRs in our repositories and pings assigned reviewers
+who have not yet approved the PR or explicitly requested changes.
+
+This DAG runs daily and pings on the following schedule based on priority label:
+
+| priority | days |
+| --- | --- |
+| critical | 1 day |
+| high | >2 days |
+| medium | >4 days |
+| low | >7 days |
+
+The DAG does not ping on Saturday and Sunday and accounts for weekend days
+when determining how much time has passed since the review.
+
+Unfortunately the DAG does not know when someone is on vacation. It is up to the
+author of the PR to re-assign review if one of the randomly selected reviewers
+is unavailable for the time period during which the PR should be reviewed.
+"""
+
+from datetime import datetime, timedelta
+
+import jinja2
+from airflow.models import DAG, Variable
+from airflow.operators.python import PythonOperator
+from common import pr_review_reminders
+from common.constants import DAG_DEFAULT_ARGS
+
+
+DAG_ID = "pr_review_reminders"
+MAX_ACTIVE_TASKS = 1
+DRY_RUN = Variable.get("PR_REVIEW_REMINDER_DRY_RUN", default_var=True)
```

Comment:
By defaulting this to `True`, I worry that we might set a precedence of having to disable dry run on other similar DAGs in the future. Do you think we could default the value based on the environment? E.g.
```suggestion
ENVIRONMENT = Variable.get("ENVIRONMENT, default_var="dev")
DRY_RUN = Variable.get("PR_REVIEW_REMINDER_DRY_RUN", default_var=(ENVIRONMENT == "dev"))
```

That way we can use `PR_REVIEW_REMINDER_DRY_RUN` locally if we'd like and override it in production when necessary, but by default will not dry run in production once deployed.

Reply:
Ah good call. I was just trying to find an easy way to run it with a PAT but also not send the comments. Getting it based on the environment makes sense.

Can you `==` against an Airflow Variable object, BTW? Cause it doesn't just return the regular string value itself, does it?

Reply:
You can! Airflow returns whatever's in the variable (rather than, say, a custom `Variable` object). You also have the option to [deserialize JSON values](https://airflow.apache.org/docs/apache-airflow/stable/concepts/variables.html), so you can get back pythonic representations of ints/floats/bools/dicts/lists/etc. 😄